tests/nested/manual/core20-install-device-file-install-ubuntu-save-via-hook/task.yaml

```diff
@@ -0,0 +1,143 @@
+summary: Verify that ubuntu-save is available during install-device hook with specific device
+
+details: |
+    This test checks that the workaround put in place for a specific device
+    continues to work. This specific device needs to install configuration files
+    onto the run system ubuntu-data on UC20 during install mode to avoid an
+    additional reboot during the first boot process.
```

Comment:
this description doesn't seem to match what is being tested

Reply:
Nice catch! Updated this